core/controllers/base_test.py

```diff
@@ -712,6 +794,43 @@ def test_controller_class_names(self):
         self.assertGreater(num_handlers_checked, 150)
 
 
+class IframeRestrictionTests(test_utils.GenericTestBase):
+
+    class MockHandler(base.BaseHandler):
```

Comment:
Give this MockHandler class a name that characterizes it better, and makes it obvious to the reader why we need it to be a mock (and why we can't just use an existing handler). E.g. MockHandlerForTestingPageIframing

Reply:
Done